journals/apps/api/permissions.py

```diff
@@ -0,0 +1,16 @@
+"""
+Permissions for apps.api views
+"""
+
+from rest_framework import permissions
+
+
+class UserPageVisitPermission(permissions.BasePermission):
+    """
+    Give permission whether user is allowed to see the page visit for a specific user.
+    """
+
+    def has_permission(self, request, view):
+        return (
+            request.user.is_staff or request.user.id == int(view.kwargs['user_id'])
+        )
```

Comment:
@bill-filler I have added the permission class according to your feedback. If you still have any concern let me know otherwise kindly merge his PR.

Reply:
I have added the concerned test case too.